src/Easing.ts

```diff
@@ -195,6 +195,30 @@ const Easing = {
 			return Easing.Bounce.Out(amount * 2 - 1) * 0.5 + 0.5
 		},
 	},
+	generatePow: function (
+		power = 4,
+	): {
+		In(amount: number): number
+		Out(amount: number): number
+		InOut(amount: number): number
+	} {
+		power = power < 1.0 ? 1.0 : power
```

Comment:
Just curious, why not allow numbers less than 1?

Reply:
I'm imagining that with a number less than 1, for example `generatePow(0.3).InOut`, the user can animate as if some fast-moving object comes into the view, then the "film speed" is slowed down and the object is in slow motion, then the "film speed" is sped back up to normal speed and the moving object takes off out of the view at full speed.

With a more exaggerated number like `0.1`, it could be as if a "UFO" was caught on camera: it comes into view and slows down very quickly, then suddenly accelerates out of view.

Reply:
@trusktr 
Thanks for your advice.

I had misunderstood that easing breaks when power is below 1. 
https://www.desmos.com/calculator/in8brckvcq
After checking, I found a beautiful curve in range of 0.0 < power < 1.0.
Pow easing curve breaks down when power is below zero.

I will modify conditions to 0.0 < power.

Reply:
nice!